packages/api-clients/open-api/bffApi.yml

```diff
@@ -20419,12 +20422,8 @@ components:
           $ref: "#/components/schemas/ClientKind"
       required:
         - id
-        - organization
```

Comment:
the frontend does not have a use case when a `CompactClient` should be returned.
Can we force a complete client here?